README.md

```diff
@@ -19,119 +19,91 @@
-
-```sh
-npm install && npm run build
-```
-
-### 2. Run Docker
+## Quickstart
 
-Find and open Docker. When you run your application, it will use Docker to host your application inside a container.
+Vectara Answer comes packaged with preset configurations that allow you spin up a sample application using Vectara's public datastores. To quickly get started, run the following command:
 
-### 3. Set configuration
+`npm run bootstrap`
 
-Duplicate the `secrets.example.toml` file and rename the copy to `secrets.toml`.
+When prompted for which application to create, simply select one of the three default apps, and you'll have the app running in your browser at `http://localhost:4444`.
 
-Edit the `secrets.toml` file and change the `api_key` value to be your Vectara API Key.
+Congratulations! You've just setup and run a sample app powered by Vectara! We'll work on setting up a custom application later in this doc.
 
-Make a duplicate of the `config/vectara-website-search/` directory and rename it `pg-search/`.
+### Under the hood
 
-Update the `config/pg-search/config.yaml` file with these changes:
+The bootstrap command installs dependencies, runs the configuration script, and spins up the local application.
 
-- Change the **corpus_id** value to the ID of the corpus into which you ingested Paul Graham's essays as part of the [`vectara-ingest` Quickstart](https://github.com/vectara/vectara-ingest/blob/main/README.md#quickstart).
-- Change the **account_id** value to the ID of your account. You can click on your username in the top-right corner to copy it to your clipboard.
-- Change the **app_title** to "Ask Paul Graham".
+If you would like to run the setup steps individually, you can run:
 
-Edit the `config/pg-search/queries.json` file and update the four questions to a set of curated questions you'd like to include in the user interface. For example: "What is a maker schedule?"
+- `npm install`: for installing dependencies
+- `npm run configure`: for running the configuration script
+- `npm run start`: for running the application locally
 
-### 4. Run the application
+### Deployment
 
-Execute the run script from the root directory using your `config/` directory, assigning the **default** profile from your secrets file:
+To set up a deployable docker image, do the following:
 
-```sh
-bash docker/run.sh config/pg-search default
-```
+- Install [Docker](https://docs.docker.com/engine/install/). Then make sure it's running on your machine.
+- Run `bash docker/run.sh`
+- View app logs with `docker logs -f vanswer`
+- Stop the app with `docker stop vanswer`
 
-The application executes inside of a [Docker container](https://www.docker.com/resources/what-container/) to avoid any issues with existing environments and installed packages.
+`run.sh` creates a deployable docker image and runs it. The app should be accessible at `http://localhost:80`.
```

Comment:
Two requests.... 1) Can we clarify the relationship with the `.env` file here?

> `run.sh` creates a deployable Docker image with the configuration from your `.env` file and runs it. The app will be accessible at `http://localhost:80`.

2) Can we create a section called "Legacy deployment" and link to a LEGACY_DEPLOYMENT.md file that contains the old deployments steps?

#### Legacy deployment

For steps on deploying Vectara Answer using a config directory, see [LEGACY_DEPLOYMENT.md](LEGACY_DEPLOYMENT.md).